test/client/app_runtime_service/app_runtime_service_client.go

```diff
@@ -0,0 +1,170 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package app_runtime_service
```

Comment:
I saw the code nearly the same as app_runtime.pb.go. 
Can we just use app_runtime.pb.go in app_test.go to make e2e test?

Reply:
I think this file is test all of `api server`, but `app_runtime.pb.go` is only `app server`.

Reply:
Ok, I know, `app_runtime.pb.go` can only test `grpc`, but this file can test `http`.